scripts/build/setup/termux_setup_golang.sh

```diff
@@ -1,5 +1,7 @@
 # Utility function for golang-using packages to setup a go toolchain.
 termux_setup_golang() {
+	export GOPATH="${TERMUX_COMMON_CACHEDIR}/go-path" GOCACHE="${TERMUX_COMMON_CACHEDIR}/go-build"
```

Comment:
Well we're moving them to `$TERMUX_COMMON_CACHEDIR` not `$TERMUX_TOPDIR`.
Those aren't *quite* the same.
https://github.com/termux/termux-packages/blob/4698d0dfbe46589ec402510eb8a1e8254bcecbce/scripts/build/termux_step_setup_variables.sh#L117
And also there's two versions of it?
https://github.com/termux/termux-packages/blob/4698d0dfbe46589ec402510eb8a1e8254bcecbce/scripts/setup-offline-bundle.sh#L23-L29
Which one is it then?
Since `termux_setup_golang` would be called in a build script surely it's `$HOME/.termux-build/_cache/go-{build,path}`, right?
Does it matter?
When does `scripts/setup-offline-bundle.sh` come into play?
Am I overthinking this?

Reply:
We are not. So default `~/go` and `~/.cache/go-build` are used.

Reply:
Thanks just wanted to verify.